ui/page/staking/stake_overview.go

```diff
@@ -44,7 +48,8 @@ type Page struct {
 	// and the root WindowNavigator.
 	*app.GenericPageModal
 
-	scroll *components.Scroll[*transactionItem]
+	scroll          *components.Scroll[*transactionItem]
+	scrollContainer *widget.List
```

Comment:
Just noticing now that `scrollContainer` was added in this PR. What's it used for?

Reply:
it uses for scroll all content for page, you can see it on many pages.